chat-plugins/info.js

```diff
@@ -122,7 +122,7 @@ exports.commands = {
 			for (let i = 0; i < Rooms.global.chatRooms.length; i++) {
 				let thisRoom = Rooms.global.chatRooms[i];
 				if (!thisRoom || thisRoom.isPrivate === true) continue;
-				let roomBanned = ((thisRoom.bannedIps && thisRoom.bannedIps[targetUser.latestIp]) || (thisRoom.bannedUsers && thisRoom.bannedUsers[targetUser.userid]));
+				let roomBanned = (Punishments.roombannedIps.has(thisRoom.id + '|' + targetUser.latestIp) || Punishments.roombannedUsers.has(thisRoom.id + '|' + targetUser.userid));
```

Comment:
These could be expired, there's a reason the new code doesn't look at these tables directly.